src/shims/v200_imaginary.js

```diff
@@ -0,0 +1,77 @@
+import { WebAudioModule } from '@webaudiomodules/api';
+/** @typedef {import('@webaudiomodules/api').WamDescriptor} WamDescriptor */
+/** @typedef {import('@webaudiomodules/api').WamNode} WamNode */
+/** @template T @typedef {import('@webaudiomodules/api').WebAudioModule} IWebAudioModule<T> */
+
+/** @implements {IWebAudioModule<WamNode>} */
+
+export class v200_WebAudioModule_shim {
+    /** @type {import('./v200_types').V200_imaginary.WebAudioModule} */
+    wam
+
+    constructor(wam) {
+        this.wam = wam
+    }
+
+	get isWebAudioModule() {
+		return true;
+	}
+
+	get moduleId() { return this.wam.moduleId }
+
+	get instanceId() { return this.wam.instanceId }
+
+	get descriptor() { 
+        return {...this.wam.descriptor, apiVersion: this.wam.descriptor.sdkVersion}
```

Comment:
Just an example of the "shim" being able to upgrade the old WAM response to match the new API